js/protoblocks.js

```diff
@@ -381,7 +381,13 @@ function ProtoBlock(name) {
         this.args = 2;
         this.parameter = true;
         this.dockTypes.push('numberout');
-        this.dockTypes.push('numberin');
+
+        if (this.name == 'multiply' || this.name == 'plus' || this.name == 'minus') {
```

Comment:
Rather than doing this here, it would be better to override the dock type in basicblock.js as per:
https://github.com/walterbender/musicblocks/blob/master/js/basicblocks.js#L2679